### PR TITLE
Remove relocated political foolishness.

### PR DESCRIPTION
**Pull request by yisraeldov**
This has no place in a FOSS project.

Even at the bottom of the readme.

Original PR https://github.com/f/awesome-chatgpt-prompts/pull/709

### CODE REVIEW

**Comment by ntala** [comment]
Please, just stop trying to silence him.   
We can all see,  each day, that:  
**Israel is killing tens of thousands of palestinian civilians and displacing millions more. It's a genocide. Please help defend their freedom and Palestine's right to exist.**  
If you have any problem with the fact that more and more people are conscious of that, you should really find someone else to blame.

**Comment by yisraeldov** [comment]
He removed all the comments from the original PR, so if any one is guilty of silencing...

**Review by nitzo — APPROVED** [review]

**Review by przvlprd — APPROVED** [review]

**Comment by isaacfink** [comment]
This has no place in an open source project, regardless of your cause of the month, there is a time and place for activism and this is neither, what that pr tried to do is not activism but virtue signaling and we already have social media for that, please don't pollute a good space with your garbage regardless of how convinced you are, thanks for removing it

**Review by hoomb — APPROVED** [review]

**Comment by f** [comment]
Well. **I will accept the PR as it is a curation of many people do not agree with me, just to respect their contribution** I will not insist on it.

But this should not mean that I have lost my reaction to Israeli government's constant and excessive massacre of Palestinian civilians, especially children. I strongly hate any kind of terrorism and occupation from both of the sides.

I just want peace ✌🏻

**Comment by yisraeldov** [comment]
Thank you.

While I think this is an important discussion to be had, my point was that this is not the correct forum to have it.

Dec 26, 2023 9:32:50 PM Fatih Kadir Akın ***@***.***>:

**Comment by alioguzhan** [comment]
> Well. **I will accept the PR as it is a curation of many people do not agree with me, just to respect their contribution** I will not insist on it.
> 
> But this should not mean that I have lost my reaction to Israeli government's constant and excessive massacre of Palestinian civilians, especially children. I strongly hate any kind of terrorism and occupation from both of the sides.
> 
> I just want peace ✌🏻

Sevgili @f, 

It was a great act to put that message here. But this... I would expect more than this from you. There are literally thousands of open-source projects and websites that put Ukraine-related banners. Some of them are still keeping it.  If it is not political how this is?

This is your repository. And that banner echoes your ideas. It shows where you do (or did) stand. This project still serves its purpose with or without that banner. You didn't reject PRs from people who stand against Palestine.  Nothing like that happened. 

`Many people not agreeing with you` is not a valid cause. Not ever. Please.   

Normally I would open a PR to put that banner back. But I don't want to be childish. So, here is a website:

https://freepalestine.dev

Use it responsibly, please.

Here, the thing is about `the sparrow carrying water to quell the flames`. And building this website is me carrying the water. 

Thank you.

**Comment by yisraeldov** [comment]
> . There are literally thousands of open-source projects and websites that put Ukraine-related banners.

I would also be happy if all those projects removed those banners as well. Also any Black-lives-mater, Blue-lives-matter, etc... 

Open discourse is a great thing, but in the right context. Discussing complex political/moral issues in the code of a FOSS project just distract from what the project is trying to accomplish.

**Comment by alioguzhan** [comment]
>Open discourse is a great thing, but in the right context. Discussing complex political/moral issues in the code of a FOSS project just distract from what the project is trying to accomplish.

I agree with you. But there is no discussing complex political issues. No discussion at all. No one encouraged the repository owner to put that banner in the first place. As I said before, the existence of that message does not affect the primary goal of this project. Consider this as if someone decides to use `master` instead of `main` for the default branch.  People should have the right to make or announce some changes to their own repositories. No need to put pressure on people to remove or add those messages. 

Bottom line: If your feelings about the banner on https://react.dev do not remove that banner, it shouldn't do it here, too.
